packages/cli/src/amplify.ts

```diff
@@ -44,4 +44,7 @@ try {
   await usageDataEmitter.emitSuccess({}, metricDimension);
 } catch (e) {
   printer.log('Failed to emit usage metrics', LogLevel.DEBUG);
+  if (e instanceof Error && e.stack) {
```

Comment:
We can log the error name and message as well

Reply:
should we get `format(error:Error)` ?

Reply:
probably either that or a `printError` function in the cli package

Reply:
I think I like the format suggestion better

Reply:
We do this a several places in our codebase, having consistent error logs would be good. Where should this format(error: Error) live?

Reply:
yes